src/DucksNet.Domain/Model/Employee.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DucksNet.Domain.Model;
+public class Employee
+{
+    public Employee(Guid idSediu, string surname, string firstName, string address, string ownerPhone, string ownerEmail)
```

Comment:
Please keep the domain rich. Using Result with factory methods and private constructors. Also please don't pass Guids in the constructor, mark them nullable instead.